storage/cost-analysis/storage_soft_delete_relative_cost_analyzer.py

```diff
@@ -31,7 +31,7 @@
 
 import argparse
 import json
-from typing import Dict, List
+from typing import Dict, List, Union
```

Comment:
As per [PEP-585](https://peps.python.org/pep-0585/), we can use `dict` and `list` directly (like `dict[str, float]`), so we can get rid of this whole import.